tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -0,0 +1,93 @@
+@api @skipOnOcV10
+Feature: get users
+  As an admin
+  I want to be able to retrieve user information
+  So that I can see the information
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And the administrator has given "Alice" the role "Admin" using the settings api
+
+
+  Scenario: admin user gets the information of a user
+    Given user "Brian" has been created with default attributes and without skeleton files
+    When user "Alice" tries to get information of user "Brian" using Graph API
+    Then the HTTP status code should be "200"
+    And the user retrieve API response should contain the following information:
+      | displayName  | id        | mail              | onPremisesSamAccountName |
+      | Brian Murphy | %uuid_v4% | brian@example.org | Brian                    |
+
+
+  Scenario: non-admin user tries to get the information of a user
+    Given user "Brian" has been created with default attributes and without skeleton files
+    When user "Brian" tries to get information of user "Alice" using Graph API
+    Then the HTTP status code should be "401"
+    And the last response should be an unauthorized response
+
+
+  Scenario: admin user gets all users
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Brian    |
+      | Carol    |
+      | David    |
```

Comment:
IMO we don't need to create these two users, `Carol` and `David`. what do you say? We can check for `Admin` and `Alice` users.